server/src/main/kotlin/suwayomi/tachidesk/manga/impl/backup/proto/ProtoBackupExport.kt

```diff
@@ -62,7 +62,8 @@ object ProtoBackupExport : ProtoBackupBase() {
                     timeOfDay
                 )
             },
-            ::scheduleAutomatedBackupTask
+            ::scheduleAutomatedBackupTask,
+            ignoreInitialValue = false
```

Comment:
does this work?
I think I started the interval in the ServerSetup because the ProtoBackupExport never gets loaded unless a backup gets created

Reply:
It should, objects are statically initialized

Reply:
Though I see now that they are initialized in ServerSetup, so I'll revert these ones

Reply:
I think I confused it with the Updater, because that only gets initialized lazyile, so I just didn't change it like this when I added the server setting subscription